src/Adapter/Alias/Repository/AliasRepository.php

```diff
@@ -80,12 +80,17 @@ public function create(string $searchTerm, array $aliases): array
         $aliasIds = [];
 
         foreach ($aliases as $searchAlias) {
+            // As search term is not a primary key, we need st
```

Comment:
`st...` what? :smile: 

though I approve the idea, throwing exceptions everytime alias exists would be too complicated considering the UX is not well adapted for that and the page isn't used that much